source/php/AcfFields/json/mhbooking-module.json

```diff
@@ -1,47 +1,80 @@
-[{
-    "key": "group_620f6978b127c",
-    "title": "Settings for Booking Service",
-    "fields": [
-        {
-            "key": "field_620f6984f8b48",
-            "label": "Endpoint for Mitt Helsingborg Booking Service",
-            "name": "mod_mhbooking_endpoint",
-            "type": "url",
-            "instructions": "You can get the endpoint from the Mitt Helsingborg Team.",
-            "required": 1,
-            "conditional_logic": 0,
-            "wrapper": {
-                "width": "",
-                "class": "",
-                "id": ""
+[
+    {
+        "key": "group_620f6978b127c",
+        "title": "Settings for Booking Service",
+        "fields": [
+            {
+                "key": "field_620f6984f8b48",
+                "label": "Endpoint for Mitt Helsingborg Booking Service",
+                "name": "mod_mhbooking_endpoint",
+                "type": "url",
+                "instructions": "You can get the endpoint from the Mitt Helsingborg Team.",
+                "required": 1,
+                "conditional_logic": 0,
+                "wrapper": {
+                    "width": "",
+                    "class": "",
+                    "id": ""
+                },
+                "default_value": "",
+                "placeholder": ""
             },
-            "default_value": "",
-            "placeholder": ""
-        }
-    ],
-    "location": [
-        [
             {
-                "param": "post_type",
-                "operator": "==",
-                "value": "mod-mhbooking"
-            }
-        ],
-        [
+                "key": "REPLACE1",
```

Comment:
How do we genereate the key here?

- [ ] Fix

Reply:
Do we have an answer for this?

Reply:
I interpreted it as any sting is ok. I change it to something more relevant.